css-layout-classes/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div class="container">
+  <header >
+    <h1 class="full-column">Choose Your Character!</h1>
+  </header>
+  <div class="row">
+    <img src="images/link.png" alt="Link" class="column-one-third">
```

Comment:
instead of putting the column class on the images directly, you want to wrap the images in a `div` and put the class on that, so you separate the `column container` from the `content`. 

so, the same setup you used on lines 27 to 38 for the column one fourth classes. 

columns work best when you have a container which is separate from the content itself. In this case, setting the image width to 100% will make it fit the container properly